main/virt/src/scripts/certconf

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Fix the  KEY_VALUES_MISMATCH error produced when the CA module is configured
+# The issue is produced because when generated the CA private key, it is copied
+# as /var/lib/zentyal/conf/ssl/ssl.key but the certificate corresponding to the
+# new key isn't issued 
+
+set -x
```

Comment:
Hi @flopezgr,

I guess you forgot to remove this debug...
In addition, please, check the following link [(Coding Style)](https://wiki.zentyal.org/wiki/Coding_Style) this allow us to accomplish a minimal code quality standards.